src/shell/store/content.js

```diff
@@ -406,7 +406,15 @@ export function saveItem(itemZUID, action = "") {
     if (item.web.metaDescription) {
       item.web.metaDescription = item.web.metaDescription.slice(0, 160);
     }
-
+    if (item.web.metaTitle) {
+      item.web.metaTitle = item.web.metaTitle.slice(0, 150);
```

Comment:
why is this being sliced at 150 characters?

Reply:
My basis is on their UI like how the metadescription and other fields were slice.
![image](https://github.com/zesty-io/manager-ui/assets/44116036/003f88d1-342e-438e-98dd-ccc2542efe50)